search-service/src/main/kotlin/com/egm/stellio/search/csr/model/Mode.kt

```diff
@@ -0,0 +1,21 @@
+package com.egm.stellio.search.csr.model
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+enum class Mode(val string: String) {
```

Comment:
not fan of this `string` name

Reply:
i tried naming it mode. 
but to get the string associated with exclusive it is : Mode.EXCLUSIVE.mode
i prefer Mode.EXCLUSIVE.string.
maybe it can be "name"

Reply:
`name` is better (or `key`?)